### PR TITLE
Update vocab to match glossary

### PR DESCRIPTION
**Pull request by carterbs**
**Description of change**
- changed e2o to electron where appropriate.
- made e2o the default container.

**Description of testing**
- ran npm run dev

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -286,13 +286,13 @@
 			const CLI_VERSION = require(path.join(CLI_PATH, "package.json")).version;
 			const CONTROLS_PATH = path.join(__dirname, "node_modules", "@chartiq", "finsemble-react-controls");
 			const CONTROLS_VERSION = require(path.join(CONTROLS_PATH, "package.json")).version;
-			
-			// Check e2o version
+
+			// Check electron version
```
Comment (NinjaManatee) [inline gulpfile.js]: This should be "electron adapter" instead of "electron" for accuracy.

File: gulpfile.js [hunk 2]
```diff
@@ -286,13 +286,13 @@
...
 			const E2O_PATH = path.join(__dirname, "node_modules", "@chartiq", "e2o");
 			const E2O_PATH_EXISTS = fs.existsSync(E2O_PATH);
-			const USING_E2O = channelAdapter === "e2o";
-			if (USING_E2O && !E2O_PATH_EXISTS) {
-				throw "Cannot use e2o channelAdapter unless e2o optional dependency is installed. Please run npm i @chartiq/e2o";
+			const USING_Electron = container === "electron";
```
Comment (NinjaManatee) [inline gulpfile.js]: Constant should be all caps. (e.g. USING_ELECTRON)

File: gulpfile.js [hunk 3]
```diff
@@ -286,13 +286,13 @@
...
+			if (USING_Electron && !E2O_PATH_EXISTS) {
+				throw "Cannot use electron container unless e2o optional dependency is installed. Please run npm i @chartiq/e2o";
```
Comment (NinjaManatee) [inline gulpfile.js]: Is the package name changing?

**Comment by NinjaManatee** [comment]
Doing a search on the seed on this branch, it looks like there are references to e2o in _gulpfile.js_ and _gulpfile-extensions-example.js_ that were not addressed. Should they have been?

File: gulpfile.js (hunk 3 above)
Comment (carterbs) [reply gulpfile.js]: At some unknown point in the very near future.

**Comment by carterbs** [comment]
> Doing a search on the seed on this branch, it looks like there are references to e2o in _gulpfile.js_ and _gulpfile-extensions-example.js_ that were not addressed. Should they have been?

@NinjaManatee They are all referencing the npm module called e2o, which will remain e2o until we rename the repo, etc.

File: gulpfile.js [hunk 4]
```diff
@@ -286,13 +285,13 @@
 			const CLI_VERSION = require(path.join(CLI_PATH, "package.json")).version;
 			const CONTROLS_PATH = path.join(__dirname, "node_modules", "@chartiq", "finsemble-react-controls");
 			const CONTROLS_VERSION = require(path.join(CONTROLS_PATH, "package.json")).version;
-			
-			// Check e2o version
+
+			// Check electron adpater version
 			const E2O_PATH = path.join(__dirname, "node_modules", "@chartiq", "e2o");
```
Comment (NinjaManatee) [inline gulpfile.js]: Do we want to leave `E2O_PATH` and `E2O_EXISTS` until we rename the package?

File: gulpfile.js [hunk 5]
```diff
@@ -286,13 +285,13 @@
...
 			const E2O_PATH_EXISTS = fs.existsSync(E2O_PATH);
-			const USING_E2O = channelAdapter === "e2o";
-			if (USING_E2O && !E2O_PATH_EXISTS) {
-				throw "Cannot use e2o channelAdapter unless e2o optional dependency is installed. Please run npm i @chartiq/e2o";
+			const USING_ELECTRON = container === "electron";
+			if (USING_ELECTRON && !E2O_PATH_EXISTS) {
+				throw "Cannot use electron container unless e2o optional dependency is installed. Please run npm i @chartiq/e2o";
```
Comment (NinjaManatee) [inline gulpfile.js]: Same as above, do we want to keep this "e2o optional dependency" until we rename the package?

File: gulpfile.js (hunk 4 above)
Comment (carterbs) [reply gulpfile.js]: yeah

File: gulpfile.js (hunk 5 above)
Comment (carterbs) [reply gulpfile.js]: I do, yes. Makes the find and replace easier.

**Pull request merged**